test/e2e/self_hosted_test.go

```diff
@@ -53,14 +53,15 @@ func TestCreateSelfHostedClusterWithBootMember(t *testing.T) {
 	os.RemoveAll(dir)
 	defer os.RemoveAll(dir)
 
+	host, _ := netutil.GetDefaultHost()
+
```

Comment:
For debugging sake, can we print this and also change "hack/test" to only run this test?